### PR TITLE
Bug/backround worker fails to cast app event

### PR DESCRIPTION
**Pull request by AndyHale**
# Bug/backround worker fails to cast to IApplicationEvent

## 📲 What

Adjust the Background Worker to use the Event Handler abstractrions in the Service Bus library.

## 🤔 Why

Using the abstractions in the CQRS.Events library is causing a cast for the message content in the Service bus library to fail, becasue it expects the interfaces in its own assembly.

## 🛠 How

Remove interfaces in the CQRS.Events library and update reference to to the Service Bus library.

## 👀 Evidence

Screenshot below shows that the service bus is receiving events now that these interfaces have been updated.

image

### CODE REVIEW

File: src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/appsettings.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
-﻿{
+{
     "OTEL_EXPORTER_OTLP_ENDPOINT": "REPLACE_ME",
     "APPLICATIONINSIGHTS_CONNECTION_STRING": "InstrumentationKey=REPLACE_ME",
 	"ServiceBusConfiguration": {
 		"Sender": {
 			"Topics": [
 				{
-					"Name": "sbt-menu-events",
```
Comment (sharonrussell) [inline src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/appsettings.json:8]: is this change needed?

**Review by sharonrussell — APPROVED** [review]
Just the one comment

File: src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/appsettings.json (hunk 1 above)
Comment (AndyHale) [reply src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/appsettings.json:8]: Appologies, no its not, that was accidentally included from a Stash.  Will revert.
